src/basics/BasicsValidator.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
```

Comment:
not needed empty lines

Reply:
> not needed empty lines

but PSR-12 requires empty line there...
And phpstorm automatically adds it while auto code-reformat